common/actions/initialize/main.js

```diff
@@ -1,16 +1,20 @@
-const utils = require('../../../utils');
+const { getInput, execStep, setFailed } = require('../../../utils');
 
 try{
-    const ghToken = utils.getInput('gh-token');
+    const npmNexusAuth = getInput('nexus-token');
```

Comment:
```suggestion
    const nexusToken = getInput('nexus-token');
```

Reply:
I think we should align names here:
![image](https://user-images.githubusercontent.com/2574275/95062662-928ae200-06fd-11eb-902f-41e1aac30ee4.png)

![image](https://user-images.githubusercontent.com/2574275/95062684-9b7bb380-06fd-11eb-9335-aeb118dfe76d.png)

https://www.marfeel.com/docs/continuous-deployment/development/nexus-repositories.html#npm-packages

Reply:
gg, it's not an auth, its a token :(

Reply:
As discussed on slack, we are going to stay with `auth` naming